### PR TITLE
release 4.1.1 -> master

### PR DESCRIPTION
**Pull request by HappyZombies**
## Summary 📖 

This simple PR merges the changes from development into master.

View the CHANGELOG for what was added and changed.

### CODE REVIEW

**Comment by HappyZombies** [comment]
I will manually publish since it's a hotfix

File: package.json [hunk 1]
```diff
@@ -17,6 +17,12 @@
     "Daniel Reguero"
   ],
   "main": "index.js",
+  "types": "index.d.ts",
+  "files": [
+    "index.js",
+    "index.d.ts",
```
Comment (jankapunkt) [inline package.json:23]: So TS types-files are from now on always bundled with the NPM package, right?

Comment (HappyZombies) [reply package.json:23]: Correct

**Review by jankapunkt — APPROVED** [review]
I give my approval with the disclaimer that I souppose you @HappyZombies know the TS types file is sufficient. I am no TS user yet.

The rest looks good / es expected.

If you still have doubts, you might choose another reviewer with focus on the TS files, I think you can also choose contributors, which are not necessarily members, too.

**Pull request merged**